md_file_standards/q01_multiple-choice/q01_multiple-choice.md

```diff
@@ -44,7 +44,8 @@ server: |
     # define correct answers
     data["correct_answers"]["part1"] = data["params"]["part1"]["ans1"]    
 part1:
- type: multiple-choice  
+ type: multiple-choice 
+ correct_answers: ans2
```

Comment:
should correct_answers be in in `pl-options`? 

rename `pl-options` to `pl-customizations`

Reply:
As predicted in #9, - this has come to bite us. This scheme will not work for dynamically generated answers (see the checkbox examples).

At this point, I see no choice but to run the python code that's supposed to be in server.py.

Reply:
Fixing in #20